### ISSUE TITLE
0.14.0 is using up 100% Disk I/O

### ISSUE THREAD

**Issue by unk1911** [bitcoin/bitcoin#9968]
This issue tracker is only for technical issues related to bitcoin-core.

General bitcoin questions and/or support requests and are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).

For reporting security issues, please read instructions at [https://bitcoincore.org/en/contact/](https://bitcoincore.org/en/contact/).

### Describe the issue
Bitcoin Core is using up 100% of Disk I/O 

### Machine specs:
- OS: Windows 8
- CPU: Intel Core i7-3770 CPU @ 3.40Ghz 
- RAM:  16GB
- Disk size:  1TB
- Disk Type (HDSDD): HD

**Comment by paveljanik** [comment]
It can't go over 100%, believe us. Since there are no details, please close this issue.

On the first run, Bitcoin Core has to sync to the network (initial block download, IBD). This phase is both CPU and disk IO intensive.

**Comment by laanwj** [comment]
Yes, this is supposed to be the case during initial sync. Normally it's set up to go as fast as possible so also will ask as much from your system as possible.

There are a few options to tweak this. For example with `-par` you can lower the parallelism of validation, if you set that to 1 (thread), it will sync slower but put less load on the system.

Also in your OS you can lower the I/O and CPU priority of the process.

**Comment by laanwj** [comment]
Closing as "expected behavior".

**Comment by unk1911** [comment]
this is not upon initial sync! this is after a full sync after it's been running for a while

**Comment by unk1911** [comment]
ok i can try tweaking those values and see what happens but may just downgrade to 0.13.2,  which did not exhibit this behavior

**Comment by TheBlueMatt** [comment]
Can you share your debug.log for the time period during which this happened? Its possible this is just because some other peer connected to you and is doing IBD from you, or, if its an SPV wallet doing it, you might try setting -peerbloomfilters=false.

**Comment by laanwj** [comment]
Reopening as it's not during initial sync.
```
you might try setting -peerbloomfilters=false.
```
Yes, forget that that is another cause of dense I/O after the initial sync: SPV wallets requesting a bloom filter scan over the block chain.

**Comment by unk1911** [comment]
i haven't seen this issue for a while now.  not sure why.  one thing is:  i'm not running a full node any more at the time (i have port 8333 blocked by the router firewall for the moment)

**Comment by unsystemizer** [comment]
Just close this issue.
And by the way, 100% doesn't mean anything. If you have a disk that does 1 IO per second, merely using it will make it "100% busy"

**Comment by laanwj** [comment]
> 100% doesn't mean anything. If you have a disk that maxes out at 1 IO per second, merely using it will make it "100% busy".

Good point. For some laptop harddisks, especially on Windows, that's pretty much reality :)

**Comment by laanwj** [comment]
Closing in favor of #9051 - having one issue open for high I/O during use (despite the OS) seems to be enough.

> i haven't seen this issue for a while now. not sure why. one thing is: i'm not running a full node any more at the time (i have port 8333 blocked by the router firewall for the moment)

As long as you run Bitcoin Core you're running a full node. Full nodes are those that keep a UTXO set and validate transactions. Blocking port 8333 doesn't have any effect on that, it will just make you unlikely to reach by bootstrapping nodes and SPV nodes won't ever connect to you so it'll indeed reduce I/O.
